lib/console.js

```diff
@@ -64,9 +64,9 @@ Console.prototype.time = function(label) {
 
 
 Console.prototype.timeEnd = function(label) {
-  var time = this._times.get(label);
+  const time = this._times.get(label);
   if (!time) {
-    throw new Error('No such label: ' + label);
+    return;
```

Comment:
Wouldn't hurt... `No such label '${label}' for console.timeEnd()` perhaps? Keep in mind, also, that the warning includes a stack trace.